### ISSUE TITLE
LeMUR `answerFormat`

### ISSUE THREAD

**Issue by smithakolan** [AssemblyAI/assemblyai-java-sdk#57]
`answerFormat` is needed to pass a query for `lemur().actionItems`. But `LemurBaseParams` doesn't have `answerFormat` as a parameter.

Currently:
```java
var response = client.lemur().actionItems(LemurBaseParams
                .builder()
                .transcriptIds(List.of(transcript.getId()))
                .context(LemurBaseParamsContext.of("A GitLab meeting to discuss logistic"))
                .build());
```

Desired:
```java
var response = client.lemur().actionItems(LemurBaseParams
                .builder()
                .transcriptIds(List.of(transcript.getId()))
                .context(LemurBaseParamsContext.of("A GitLab meeting to discuss logistic"))
                .answerFormat("Bullet points")
                .build());
```

**Comment by Swimburger** [comment]
AFAIK, action items don't have an answer format parameter.
It's not in our documentation (and thus our spec). Is this a supported parameter we didn't document?

**Comment by dsinghvi** [comment]
Will be closing this, @smithakolan do you want to reopen if you find it to be in the docs?

**Comment by smithakolan** [comment]
@Swimburger https://www.assemblyai.com/docs/lemur/specialized-endpoints#action-items

I'm seeing `answer format` as a param for both the TS & Python versions of the action items code, unless those are outdated?

**Comment by Swimburger** [comment]
@smithakolan, it looks like the API ref for LeMUR was missing this for action items, and thus the spec also missed them.
It's not legal ATM for the TS SDK, but it should be allowed.

I'll update the spec and SDKs.

**Comment by Swimburger** [comment]
Answer format is included in 1.0.6
